lib/classes/PluginManager.test.js

```diff
@@ -1438,7 +1443,9 @@ describe('PluginManager', () => {
       return pluginManager.run(commandsArray).then(() => {
         expect(consoleLogStub.called).is.equal(true);
         pluginManager.serverless.cli.log.restore();
-        process.env.SLS_DEBUG = undefined;
+      }).finally(() => {
```

Comment:
Is this the preferred way to do cleanup? I always love `afterEach` or `afterAll` becaus this keeps it consistent and makes it easier to understand the setup / teardown process.

Reply:
In that case it's only single test out of 10+ that depends on it, and I'm not sure is it nice to create dedicated describe here, and mangling this env var for other tests might not turn desired.

After we drop Node.js v4 support, I can update it to something as:

```javascript
it('....', overrideEnv(() => {
  process.env.SLS_DEBUG = '*';
  // test logic
}));
```